### PR TITLE
atlas 13tev ttb hadr implementation

### PR DESCRIPTION
**Pull request by t7phy**
~~https://vp.nnpdf.science/IsNUvaBuRLuWHFQ5CjPpVg==~~
https://vp.nnpdf.science/aMe4tajeTqG1pruPI5iI8g==

### CODE REVIEW

**Comment by scarlehoff** [comment]
I think there might be some info missing or a bug somewhere in the reader since there's no label in the y axis https://vp.nnpdf.science/IsNUvaBuRLuWHFQ5CjPpVg==/#matched_datasets_from_dataspecs0_plot_fancy_dataspecs_0

(ex. of dataset with y label https://vp.nnpdf.science/oSZLrPg3Tyyf-i2mE33G-Q==/#dat_compare3_dataset_inputs0_plot_fancy_0)

Also, what does `impl` mean?

**Comment by Radonirinaunimi** [comment]
Aren't you just missing the key `y_label` inside `plotting`?

**Comment by t7phy** [comment]
> Aren't you just missing the key `y_label` inside `plotting`?

I didn't even know we had a y_label in plotting block. Shouldn't the y axis label come from observable label ?

**Comment by Radonirinaunimi** [comment]
> > Aren't you just missing the key `y_label` inside `plotting`?
> 
> I didn't even know we had a y_label in plotting block. Shouldn't the y axis label come from observable label ?

You mean `dataset_label`?. They don't have to be necessarily the same. `dataset_label` is a more generic description and appears as title in the plot while `y_label` exactly specifies what's on the `y`-axis, as you can see on the second example above.

**Comment by t7phy** [comment]
> > > Aren't you just missing the key `y_label` inside `plotting`?
> > 
> > 
> > I didn't even know we had a y_label in plotting block. Shouldn't the y axis label come from observable label ?
> 
> You mean `dataset_label`?. They don't have to be necessarily the same. `dataset_label` is a more generic description and appears as title in the plot while `y_label` exactly specifies what's on the `y`-axis, as you can see on the second example above.

No what I meant is as follows: in any of the implementation, I never added x_label or y_label in the plotting block. For x_label it always took from the label of the particular kinematic variable used, so for y_label, I assumed it would take from observable label (there is a observable key, which contains 3 more keys: description, label and units, so I am referring to that observable label). but anyway, I guess y_label will be needed in plotting block, but this will need a fix in all datasets that I implemented then.

**Comment by t7phy** [comment]
> Yes, otherwise it'll never work.

yes, however I think this fix would be best done in a separate PR because it needs to be done for all the datasets, and I will do that asap, so for the time being, can we atleast merge this implementation?

File: validphys2/src/validphys/datafiles/new_commondata/ATLAS_TTBAR_13TEV_HADR_DIF/metadata.yaml
Comment (Radonirinaunimi) [inline validphys2/src/validphys/datafiles/new_commondata/ATLAS_TTBAR_13TEV_HADR_DIF/metadata.yaml:93]:
```suggestion
      y_label: '$\frac{d\sigma}{d|y_{t\bar{t}}|}$ ($pb$)'
```

**Review by Radonirinaunimi — CHANGES_REQUESTED** [review]
Apart from the typo below, this looks good to me.

Later on, at some point, we should take some decisions regarding the utils from which more and more dataset implementations rely on.

**Review by Radonirinaunimi — APPROVED** [review]

**Comment by Radonirinaunimi** [comment]
@t7phy, for future reference, before this PR gets merged, could you please update the report in the description? Thanks.

**Comment by t7phy** [comment]
> @t7phy, for future reference, before this PR gets merged, could you please update the report in the description? Thanks.

done! I will merge once the tests complete.

**Pull request merged**
